src/main.js

```diff
@@ -40,6 +40,12 @@ module.exports = {
       type: 'string',
       default: ''
     },
+    localNodeModules: {
+      title: 'Path to the local node_modules folder',
+      description: 'Optionally specify the path to the local node_modules folder',
```

Comment:
Maybe put `, can be relative to the current project` on the end so users know about that?